collector/otel-setup.mdx

```diff
@@ -0,0 +1,56 @@
+---
+title: 'Set up OpenTelemetry exporter with the pganalyze Collector'
+backlink_href: /docs/collector
+backlink_title: 'pganalyze Collector'
+---
+
+To integrate with application performance monitoring (APM) tools, the collector
+can optionally act as an [OpenTelemetry tracing](https://opentelemetry.io/docs/concepts/signals/traces/)
+exporter.
+This adds data from pganalyze into existing application traces, and allows
+application engineers to have a quick way to find out why SQL queries are slow.
+
+The exporter will send a tracing span for each [collected EXPLAIN plan](/docs/explain/setup)
+to the configured OpenTelemetry endpoint, with a link back to the full plan in
+pganalyze via the `db.postgresql.plan` span attribute.
+The span will be recorded with a service name "Postgres (pganalyze)" and a span
+name "EXPLAIN Plan" by default.
+
+![Overview of OpenTelemetry setup with the pganalyze collector](otel-overview.svg)
+
+## Setup the pganalyze collector with third-party services
+
+By setting up `otel_exporter_otlp_endpoint` (or `OTEL_EXPORTER_OTLP_ENDPOINT`),
+the collector will start sending out tracing spans to the specified endpoint.
+
+You can test this by pointing this endpoint to an OpenTelemetry collector
+instance running locally too. For example, you can use the [collector Docker image](https://hub.docker.com/r/otel/opentelemetry-collector-contrib)
```

Comment:
Yeah good point. I think I only mention the OpenTelemetry collector here, so I'll make it extra clear about what I'm talking about.

Reply:
Actually I was thinking about making this as "Setting up with the OpenTelemetry collector" or "Setting up locally".
Maybe having a separate page will help clarifying (so that only that page will talk about the OpenTelemetry collector). I'll think a bit more.

Reply:
Ok I didn't make a new page but I put this into the separate section, "### Test setting up locally with the OpenTelemetry collector". I think it's better as a structure, though I'm not super sure if this is even needed.
This is really about test/dev env and feels like this much testing is not really required by customers (maybe only us).
I'm fine dropping the whole section too if it can be too much (or too confusing). Thoughts? @lfittl

Reply:
Yeah, this section is now very clear. Thanks for rewriting.